src/gigs-board/components/community/CommunityHeader.jsx

```diff
@@ -74,108 +77,109 @@ const BreadcrumbLink = styled.a`
     text-decoration: none;
   }
 `;
+
 const BreadcrumbBold = styled.b`
    {
     color: #3252a6;
   }
 `;
 
-// TODO nav-underline is available in bootstrap: https://getbootstrap.com/docs/5.3/components/navs-tabs/#underline,
-// but it's not there in near social, need write such style here
-return (
-  <>
-    <Header>
-      <div aria-label="breadcrumb">
-        <ol class="breadcrumb">
-          <li class="breadcrumb-item">
-            <BreadcrumbLink href={href("Feed")}>DevHub</BreadcrumbLink>
-          </li>
-          <li class="breadcrumb-item active" aria-current="page">
-            <BreadcrumbBold>{props.title}</BreadcrumbBold>
-          </li>
-        </ol>
-      </div>
-      <div class="d-flex flex-row align-items-center pb-3">
-        <img src={props.icon} width="95px" height="95px"></img>
-        <div>
-          <div class="h5 pt-3 ps-3">{props.title}</div>
-          <div class="ps-3 pb-2 text-secondary">{props.desc}</div>
+const topicTabs = [
+  {
+    defaultActive: true,
+    iconClass: "bi bi-house-door",
+    path: "community.Overview",
+    title: "Overview",
+  },
+  {
+    iconClass: "bi bi-chat-square-text",
+    path: "community.Discussions",
+    title: "Discussions",
+  },
+  {
+    iconClass: "bi bi-coin",
+    path: "community.Sponsorship",
+    title: "Sponsorship",
+  },
+  {
+    iconClass: "bi bi-calendar",
+    path: "community.Events",
+    title: "Events",
+  },
+  {
+    iconClass: "bi bi-github",
+    path: "community.github",
+    title: "GitHub",
+  },
```

Comment:
```suggestion
  },
  {
    iconClass: "bi bi-telegram",
    path: "community.Telegram",
    title: "Telegram",
  },
```

Telegram is merged to main, so adding this tab as well.